y_pef_schedule.routing.yml

```diff
@@ -0,0 +1,67 @@
+y_pef_schedule.branches:
+  path: '/admin/openy/branch-schedules'
+  defaults:
+    _controller: '\Drupal\y_pef_schedule\Controller\FullCalendarController::branchesListPage'
+    _title: 'Branches'
+  requirements:
+    _permission: 'access administration pages'
+
+y_pef_schedule.fullcalendar:
+  path: '/admin/openy/branch-schedules/{branch}'
+  defaults:
+    _controller: '\Drupal\y_pef_schedule\Controller\FullCalendarController::calendarView'
+    _title: 'Schedules'
+    branch: ''
+  requirements:
+    _permission: 'access administration pages'
+
+y_pef_schedule.branches_options:
+  path: '/branches-options'
+  defaults:
+    _controller: '\Drupal\y_pef_schedule\Controller\FullCalendarController::getBranchesOptions'
+    _title: 'Schedules'
+  requirements:
+    _permission: 'access administration pages'
+
+y_pef_schedule.classes_options:
+  path: '/classes-options'
+  defaults:
+    _controller: '\Drupal\y_pef_schedule\Controller\FullCalendarController::getClassesOptions'
+    _title: 'Schedules'
+  requirements:
+    _permission: 'access administration pages'
+
+y_pef_schedule.create_event:
+  path: '/admin/openy/branch-schedules/create-event'
+  defaults:
+    _controller: '\Drupal\y_pef_schedule\Controller\FullCalendarController::createEvent'
+    _title: 'Create event'
+  requirements:
+    _permission: 'access administration pages'
+
+y_pef_schedule.update_event:
+  path: '/admin/openy/branch-schedules/update-event'
```

Comment:
if we can avoid /admin/ for these, please
#security

Reply:
Done.
The admin path is saved only for the list of branches and the calendar itself, if they also need to be changed, please let me know.